### DISCUSSION TITLE
Small issue with trigger=True textboxes

### DISCUSSION THREAD

**Discussion by Far0n** [h2oai/wave#948] [General]
1) `trigger=True` on textboxes feels a bit clunky, because a) focus gets lost after submit and b) submit might happen if we type too slowly.  Example: https://wave.h2o.ai/docs/examples/textbox-trigger/
2) demo is prone to crash (see below)

https://user-images.githubusercontent.com/6949295/128642343-2062281c-2439-457b-902a-b6f88ae9f902.mp4

**Comment by mturoci** [comment]
Hi @Far0n

1 a) - https://github.com/h2oai/wave/issues/150
1 b) - this is expected, there is `500ms` debounce timeout that we felt like a reasonable default.
2 - will fix

**Reply by Far0n** [reply]
Thanks for clarification @mturoci and sry regarding a) .. should have checked the open issues. Thought I did, but apparently I didn't. ^^

**Reply by mturoci** [reply]
No problem at all :) Happy to help.
